### ISSUE TITLE
Row height in produced worksheets

### ISSUE THREAD

**Issue by sampov2** [protobi/js-xlsx#25]
Is it possible to adjust row heights (per row) in worksheets written using this library?

**Comment by pietersv** [comment]
I don't believe there is but that seems an important thing to include.  Am hesitant to make further extensions until the tests are running in Node 4+ or i invest time to install multiple versions of Node.

If you're game to extend js-xlsx for your immediate purpose, row heights are set by an attribute `ht` and setting `customHeight="1"` on the `<row>` element, e,g, 

```xml
     <row r="1" spans="1:3" ht="100" customHeight="1">
```

If the goal is simply generate a new xlsx file, [chuanyi/msexcel-builder](https://github.com/chuanyi/msexcel-builder) does have row height, and the [protobi/msexcel-builder](https://github.com/chuanyi/msexcel-builder) of it has some extensions for styles, autofilter, etc. just not yet as complete as js-xlsx.

**Comment by toby-howell** [comment]
>Am hesitant to make further extensions until the tests are running in Node 4+

Did https://github.com/SheetJS/js-xlsx/commit/2a756fffccc30bf6871d7f04ede61a6fef160555 get those tests passing?

**Comment by codecounselor** [comment]
I was going to offer up a PR but I've got some other changes that are causing conflicts.  If anyone is interested you can cherry-pick this commit or just copy the changes wholesale into your work. 
https://github.com/SpiderStrategies/js-xlsx/commit/0d1f7a080f1a9848a2ef9f9f7cd50d149ff2eb55
